### PR TITLE
Added a few classes util for structure relaxations (in particular MOMONANO)

### PR DESCRIPTION
**Pull request by jnsLs**
- added batch-wise atoms converter
- modified ase_interface (for optimization)
- added neighborlist wrapper (for postprocessing of the nbh list, e.g., neglecting neighbors)
- adapted Skin NBL documentation

### CODE REVIEW

File: src/schnetpack/transform/neighborlist.py [hunk 1]
```diff
@@ -29,6 +31,37 @@ class CacheException(Exception):
     pass
 
 
+class NeighborlistWrapper(Transform):
```
Comment (ktschuett) [inline src/schnetpack/transform/neighborlist.py:34]: why is this needed? why not add a separate preprocessing layer?

File: src/schnetpack/transform/neighborlist.py [hunk 2]
```diff
@@ -480,6 +506,42 @@ def _get_shifts(self, cell, pbc, cutoff):
         )
 
 
+class RemoveSomeNeighbors(Transform):
```
Comment (ktschuett) [inline src/schnetpack/transform/neighborlist.py:509]: bad name (some?). FilterNeighbors?

**Review by ktschuett — CHANGES_REQUESTED** [review]

File: src/schnetpack/transform/neighborlist.py (hunk 1 above)
Comment (jnsLs) [reply src/schnetpack/transform/neighborlist.py:34]: for the training procedure you are right: one could just use a separate preprocessing layer. However, in the scope of MD and structure relaxation, this wrapper makes it more convenient to use NBL postprocessing. For example, the AtomsConverter in its current state does not allow the usage of additional pre/postprocessing. One would either have to implement a postprocessing parameter there or keep it as is and use the nbl wrapper instead.

**Review by ktschuett — APPROVED** [review]
